infrastructure/versions.tf

```diff
@@ -5,17 +5,24 @@ terraform {
       source  = "hashicorp/azurerm"
       version = "~> 3.5.0"
     }
+
+    helm = {
+      source  = "helm"
+      version = ">= 2.4.1"
+    }
+
+    kubernetes = {
+      source  = "kubernetes"
+      version = ">= 2.8.0"
```

Comment:
Is this the latest version of k8's or AKS? I believe the latest version for AKS is 
1.22.6 on azure, of course I might be wrong :)